audit/audit.go

```diff
@@ -34,8 +37,24 @@ func init() {
 	if flag.Lookup("test.v") != nil {
 		return
 	}
+	var auditErrors []error
+	re := regexp.MustCompile("postgres://(.*?):(.*?)@(.*)/(.*)")
```

Comment:
I'm not sure we should perform this validation. In any case, I think this regex is wrong, because `postgres://localhost/db` should work (it should expand to `postgres://current_user@localhost:5432/db`).

I think it might be easier to just let the postgres library handle this. If the user provides an invalid connection string, it'll return an error that we can just display.

Reply:
agree with that, but not matching regex will just print connection string as is, the only reason to have this regex is somehow to tell user which connection is failing and mask the password, assuming we have couple of databases defined. If I shouldn't worry about this I will happy to omit this part.

Reply:
If we're worried about that, I think maybe getting the error message returned by libpq and replacing the connection string with `<connection string redacted>` or something along those lines would be better. But I don't think showing the connection string on stderr is a big deal, so I don't think we need this.